cppcache/integration/test/CMakeLists.txt

```diff
@@ -117,7 +114,7 @@ file(GLOB SSL_CERTIFICATES
         "${CMAKE_CURRENT_SOURCE_DIR}/*Keys"
         )
 
-file(INSTALL ${SSL_CERTIFICATES} DESTINATION ${CMAKE_CURRENT_BINARY_DIR})
+#file(INSTALL ${SSL_CERTIFICATES} DESTINATION ${CMAKE_CURRENT_BINARY_DIR})
```

Comment:
Still missed this commented out line - please delete.

Reply:
Weird. Not sure how that came back.